doc/source/development/notebooks/processor_examples/api.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you rename this file such that it is clear that it belongs to the `requester.ipynb` notebook? Maybe `requester_api_example.json`.

Reply:
done